docs/source/core.rst

```diff
@@ -83,37 +83,115 @@ useBlockNumber
 
 Get the current block number. Will update automatically when the new block is mined.
 
-useChainCall
-============
+useCall
+=======
+
+Makes a call to a specific contract and returns the value or an error if present. The hook will cause the component to refresh when a new block is mined and the return value changes.
```

Comment:
...specific function of a specific contract...

Reply:
✅